src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -307,6 +308,11 @@ internal void ValidateOptions(DiagnosticBag diagnostics, CommonMessageProvider m
             {
                 diagnostics.Add(messageProvider.CreateDiagnostic(messageProvider.ERR_InvalidHashAlgorithmName, Location.None, ""));
             }
+
+            if (PdbFilePath != null && !PathUtilities.IsValidFilePath(PdbFilePath))
+            {
+                diagnostics.Add(messageProvider.CreateDiagnostic(messageProvider.FTL_InputFileNameTooLong, Location.None, PdbFilePath));
```

Comment:
>FTL_InputFileNameTooLong [](start = 81, length = 24)

We probably should use name that is more accurately reflects the message. For example, FTL_InvalidInputFileName.

Reply:
Let's open a separate issue to follow up to avoid possible extra work for LOC team in dev15.7.x.

---
In reply to: [179337190](https://github.com/dotnet/roslyn/pull/25903#discussion_r179337190) [](ancestors = 179337190)

Reply:
Created #25944

---
In reply to: [179337455](https://github.com/dotnet/roslyn/pull/25903#discussion_r179337455) [](ancestors = 179337455,179337190)